core/api/jrpc/value_converter.hpp

```diff
@@ -34,6 +39,15 @@ namespace kagome::api {
     }
     return value;
   }
+
+  template <class T1, class T2>
```

Comment:
Think about to implement variadic template for variant with more then two types

Reply:
I tried, but it would take more time and it's not clear if we need more than 2 types. Maybe in the future.